Source/Core/Row.swift

```diff
@@ -50,45 +49,90 @@ open class RowOf<T: Equatable>: BaseRow {
             }
         }
     }
-    
+
+    /// The typed value of this row.
+    public var value : T?{
+        set (newValue){
+            _value = newValue
+            if validationOptions.contains(.validatesOnChange) || (blurred && validationOptions.contains(.validatesOnChangeAfterBlurred)) || !isValid   {
```

Comment:
if the row is invalid we want to run validations on each value change to show the validation passing as soon as possible to the user. 
We've talked about it before.